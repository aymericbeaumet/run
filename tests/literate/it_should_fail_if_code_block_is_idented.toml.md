# Indented code block

    [[run]]
    cmd = ["coreutils", "echo", "indented"]