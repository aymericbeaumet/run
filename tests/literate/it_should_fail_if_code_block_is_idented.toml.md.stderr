Error: failed to load the literate config file at $CARGO_MANIFEST_DIR/tests/literate/it_should_fail_if_code_block_is_idented.toml.md

Caused by:
    code block must start with ```toml (indented is not allowed)